nifi-nar-bundles/nifi-snowflake-bundle/nifi-snowflake-services/src/main/java/org/apache/nifi/snowflake/service/util/AccountIdentifierFormatParameters.java

```diff
@@ -19,19 +19,19 @@
 
 public final class AccountIdentifierFormatParameters extends SnowflakeCommonParameters {
 
-    private final String hostUrl;
+    private final String accountUrl;
 
-    public AccountIdentifierFormatParameters(final String hostUrl,
+    public AccountIdentifierFormatParameters(final String accountUrl,
```

Comment:
When I tried to understand the code, the parts related to this class were hard to read. In `AccountIdentifierFormatTest`, I already pointed out that it is a bit weird that 5/6 parameters of this class can be null. I think the root issue is that this class is an unnecessary intermediate step, at least in this form. We need to parse the properties from the context into this POJO, then in the `AccountIdentifierFormat`, we need to declare again which properties we need. The clean way would be to create `*FormatParameter` POJOs for each `AccountIdentifierFormat`. But this could be an overkill. The other option would be removing this class and extracting the required parameters from the context in the `AccountIdentifierFormat` class. Either option is good for me; the latter is probably better because the extra complexity wouldn't bring additional value.